packages/admin/cms-admin/src/common/header/UserHeaderItem.tsx

```diff
@@ -48,8 +48,30 @@ export function UserHeaderItem(props: PropsWithChildren<UserHeaderItemProps>) {
     const [showAboutModal, setShowAboutModal] = useState(false);
     const [signOut, { loading: isSigningOut }] = useMutation<GQLSignOutMutation>(signOutMutation);
 
-    const AccountIcon = user.impersonated ? <Account color="info" /> : <Account />;
-
+    const AccountIcon = user.impersonated ? (
+        <AvatarGroup>
+            <StyledAvatar
+                sx={{
+                    opacity: "50%",
+                }}
+            >
+                <Account />
+            </StyledAvatar>
+            <StyledAvatar
+                sx={{
+                    border: `2px solid ${theme.palette.primary.main} !important`,
+                    marginLeft: "-10px",
+                    zIndex: 1,
+                }}
+            >
+                <ImpersonateUser />
+            </StyledAvatar>
```

Comment:
Please avoid combining styled-components with the `sx` prop. 
You can either extend the existing styled-component for additional variants or define props for conditional styling.